src/pages/stepsCalculator/stepsCalculator.tsx

```diff
@@ -0,0 +1,148 @@
+        if (heightCm <= 0) {
+            return 0;
+        }
+    }
+
+    function placeHolder() {
+        // return steps.toString();
+        const updateSteps = document.querySelector('#result');
+        if (updateSteps != null) {
+            if (metric == false && heightFt > 0 && heightIn > 0) {
+                // imperial calculations
+                // height in IN * convert = inches per stride
+                // IN per stride / 12 = feet per stride
+                // 5280 / feet per stride = steps per mile
+                steps = miles * (5280 / ((((heightFt * 12) + heightIn) * strideConverter) / 12));
+            }
+            else if (metric == true) {
+                // metric calculations
+            }
+            updateSteps.innerHTML = Math.round(steps).toString();
+        }
+        // updateSteps.innerHTML = steps.toString();
+        return;
+    }
+
+    return (
+        <>
+            <IonHeader>
+                <IonToolbar>
+                    <IonTitle>
+                        Steps Calculator
+                    </IonTitle>
+                </IonToolbar>
+            </IonHeader>
+
+            <IonContent className="ion-padding">
+                <IonItem>
+                    <IonLabel position="floating">Number of miles walked</IonLabel>
+                    <IonInput type="number" onInput={
+                        (event: any) => {
+                            calculate(event);
+                        }
+                    }></IonInput>
+                    <IonRouterLink slot="helper" href="./manualLoggingSteps">Return to steps logging</IonRouterLink>
+                </IonItem>
+                <IonGrid>
+                    <IonRow>
+                        <IonCol>
+                            <IonItem>
+                                <IonLabel position="floating">Optional: Height ft</IonLabel>
+                                <IonInput type="number" onInput={
+                                    (event: any) => {
+                                        getHeightFt(event);
+                                    }
+                                }></IonInput>
+                            </IonItem>
+                        </IonCol>
+                        <IonCol>
+                            <IonItem>
+                                <IonLabel position="floating">Optional: Height In</IonLabel>
+                                <IonInput type="number" onInput={
+                                    (event: any) => {
+                                        getHeightIn(event);
+                                    }
+                                }></IonInput>
+                            </IonItem>
+                        </IonCol>
+                    </IonRow>
+                </IonGrid>
+                <IonCol>
+                    <IonButton onClick={placeHolder}>Submit</IonButton>
+                </IonCol>
+                <IonItem id="result">{steps.toString()}</IonItem>
```

Comment:
I'm assuming that steps.toString() would print 4,000 without the comma. Do we want to try to do some fancy formatting, or pull in a library that will do fancy formatting, to add commas into numbers? Or is my assumption wrong?

Reply:
It seems that toString does ommit commas. I can add in something pretty fast to make it have some commas where needed. I'll look into it more, and maybe see if there is a library that can be pulled that works

Reply:
Resolved, calls to toLocalString('lang-area') aka 'en-US' will make the stuff formatted to us standards in english. Such as dates being mm-dd-yyyy and numbers being nnn,nnn,nnn etc